### PR TITLE
Fx register

### PR DESCRIPTION
**Pull request by Schade9**
PR Checklist

- [ ] - Latest `master` branch merged
- [ ] - PR title descriptive (can be used in release notes)

## Description

_This fixes all endpoints for retrieving a single record from medications, medicine, resilience and seizures.
It also adds more fields to the user model as well as a blob field for the profile picture as a largeBinary_

## Git Issues

_None_

### CODE REVIEW

File: app/schemas/user.py [hunk 1]
```diff
@@ -6,6 +6,13 @@ class UserSchema(Schema):
...
     email = fields.Email()
     username = fields.String()
+    age = fields.Integer()
+    gender = fields.String()
+    age_of_onset = fields.Integer()
+    seizure_type = fields.String()
+    caregiver_name = fields.String()
+    caregiver_contact = fields.String()
+    institution = fields.String()
     password = fields.String(load_only=True)
```
Comment (MutegekiHenry) [inline app/schemas/user.py:18]: @Schade9 seems you forgot to add the profileImage field to the schema.. meaning even when its sent it won't be picked up and stored.

**Review by MutegekiHenry — COMMENTED** [review]
Great work on this @Schade9 kindly address the comment and let me know.

File: app/schemas/user.py (hunk 1 above)
Comment (Schade9) [reply app/schemas/user.py:18]: @MutegekiHenry i have added it

**Review by MutegekiHenry — APPROVED** [review]
Great work on the fixes and additions @Schade9 ! As a general comment, lets try to have separate PRs addressing granola issues like a PR to fix something specific to ease readability.

**Pull request merged**